src/go/rpk/pkg/tuners/irq/cpu_masks.go

```diff
@@ -70,7 +70,7 @@ func (masks *cpuMasks) CPUMaskForComputations(
 	mode Mode, cpuMask string,
 ) (string, error) {
 	log.Debugf("Computing CPU mask for '%s' mode and input CPU mask '%s'", mode, cpuMask)
-	var computationsMask = ""
```

Comment:
Realistically this should be `var computationsMask string`, but `:= ""` is ok too for now.